posydon/popsyn/io.py

```diff
@@ -146,15 +146,19 @@
     'f_fb': 'float64',
     'spin_orbit_tilt_first_SN': 'float64',
     'spin_orbit_tilt_second_SN': 'float64',
+    'M4': 'float64',
+    'mu4': 'float64',
+    'avg_c_in_c_core_at_He_depletion' : 'float64',
+    'co_core_mass_at_He_depletion' : 'float64',
```

Comment:
I'd suggest to move `avg_c_in_c_core_at_He_depletion` and `co_core_mass_at_He_depletion` to `STARPROPERTIES_DTYPES`.

Reply:
You put them to `EXTRA_STAR_COLUMNS_DTYPES`, why didn't you put them to `STARPROPERTIES_DTYPES`?